.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,27 @@
+name: Deploy Playground to GitHub Pages
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+permissions:
+  contents: write
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v3 
+      # this action has a built-in functionality for caching and restoring dependencies
+      # https://github.com/actions/setup-node#caching-global-packages-data
+        with:
+          node-version: 16
+      - name: Install and Build
+        run: |
+          npm ci
+          npx nx build playground
+      - name: Deploy 🚀
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: dist/packages/playground
```

Comment:
I am having some issues here. The action runs perfectly fine and the artefacts get uploaded to `gh-pages` branch. 
However, I couldn't see the site is live as expected. Not sure if I miss anything on the configuration


![Screenshot 2022-10-16 at 11 08 41 PM](https://user-images.githubusercontent.com/6767322/196043321-e72a62c2-ebdf-458b-9d26-eae577f1aac3.jpg)

Reply:
✅ Solved, used the the set of actions `actions/configure-pages`, `actions/upload-pages-artifact` and `actions/deploy-pages`